package.json

```diff
@@ -27,7 +27,8 @@
         "prop-types": "15.7.2",
         "react-csv-downloader": "^2.7.0",
         "react-request-fullscreen": "^1.1.2",
-        "react-virtualized": "^9.21.2"
+        "react-virtualized": "^9.21.2",
+        "@material-ui/icons": "^4.5.1"
```

Comment:
is it necessary ?

Reply:
because it's presents in the peerdependencies List

Reply:
removed